File: Src/ButterflyNet.Parser/Scopes/DefinitionListScope.cs
namespace ButterflyNet.Parser.Scopes {
	public class DefinitionListScope : BlockScope {
		public override bool CanNestText { get { return false; } }

		public override void Open(ButterflyAnalyzer analyzer) {
			analyzer.OpenDefinitionList();
		}

		public override void Close(ButterflyAnalyzer analyzer) {
			analyzer.CloseDefinitionList();
		}
	}
}

File: src/js/src/scopes/DefinitionListScope.js
function DefinitionListScope() {
	this.type = ScopeTypeCache.definitionList;
	this.level = ScopeLevel.block;
	
	this.open = function(analyzer) {
		analyzer.openDefinitionList();
	};
	this.close = function(analyzer) {
		analyzer.closeDefinitionList();
	};
}

extend(Scope, DefinitionListScope);